rust/chains/hyperlane-ethereum/src/mailbox.rs

```diff
@@ -282,6 +282,15 @@ where
                 tx = tx.gas_price(gas_price);
             }
         }
+        if self.domain.id() == 56 {
+            if let Ok(gas_price) = std::env::var("BSC_GAS_PRICE")
```

Comment:
is this because we don't have gas overrides on v2?

Reply:
yeah :(